### DISCUSSION TITLE
[@types/shippo] Missing messages property on the Shipment interface

### DISCUSSION THREAD

**Discussion by acommodari** [DefinitelyTyped/DefinitelyTyped#69182] [Issues with a @types package] [answered]
Hello I was working with `@types/shippo` and I noticed that the response from creating a new shipment did not include the `messages` property type. This type is important for logging purposes and it would be great to have this natively inside the types.

Here is the api reference: https://docs.goshippo.com/shippoapi/public-api/#operation/CreateShipment

[a comment by an automated account is not shown]

**Comment by clalexander** [comment, marked as answer]
@acommodari You are right that `messages` is missing, along with a few other properties. I will open a PR with the updates.

**Comment by clalexander** [comment]
Added issue #69261

**Comment by clalexander** [comment]
Opened PR #69263

**Comment by clalexander** [comment]
Resolved
